src/ios/dependencies/Branch-SDK/Branch.m

```diff
@@ -379,12 +396,13 @@ + (void) setBranchKey:(NSString*)branchKey {
                 @"You are using your test app's Branch Key. "
                  "Remember to change it to live Branch Key for production deployment."
             );
-        } else
-        if ([branchKey hasPrefix:@"key_live"]) {
+        
+        } else if ([branchKey hasPrefix:@"key_live"]) {
```

Comment:
I noticed that there are a lot of style changes. Do we have a standard style guide or checker?

Reply:
For the most part, it's up to the developer.  Though we should be following Apple's guidelines whenever possible.

Reply:
Can we enforce a style guide for consistency (at least for changes made by Branch employees)?

Reply:
I can look into it.  Haven't had the need before.